From 91fa5ef37a36c9307e4ef74b6ed3d76eaf975692 Mon Sep 17 00:00:00 2001
From: Dominique Hazael-Massieux <dom@w3.org>
Date: Mon, 2 Aug 2021 11:43:45 +0200
Subject: [PATCH] Patch Portals IDL to reference new version of PostMessage
 Options

See https://github.com/WICG/portals/pull/273 and https://github.com/WICG/portals/issues/275 (patch has been merged but published spec not updated yet)
---
 ed/idl/portals.idl | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/ed/idl/portals.idl b/ed/idl/portals.idl
index 20c655b92..a6043e4d7 100644
--- a/ed/idl/portals.idl
+++ b/ed/idl/portals.idl
@@ -11,13 +11,13 @@ interface HTMLPortalElement : HTMLElement {
     [CEReactions] attribute DOMString referrerPolicy;
 
     [NewObject] Promise<undefined> activate(optional PortalActivateOptions options = {});
-    undefined postMessage(any message, optional PostMessageOptions options = {});
+    undefined postMessage(any message, optional StructuredSerializeOptions options = {});
 
     attribute EventHandler onmessage;
     attribute EventHandler onmessageerror;
 };
 
-dictionary PortalActivateOptions : PostMessageOptions {
+dictionary PortalActivateOptions : StructuredSerializeOptions {
     any data;
 };
 
@@ -27,7 +27,7 @@ partial interface Window {
 
 [Exposed=Window]
 interface PortalHost : EventTarget {
-    undefined postMessage(any message, optional PostMessageOptions options = {});
+    undefined postMessage(any message, optional StructuredSerializeOptions options = {});
 
     attribute EventHandler onmessage;
     attribute EventHandler onmessageerror;
-- 
2.32.0

